configs/server_example.cfg

```diff
@@ -36,32 +36,31 @@
-sets " Location"                                         ""   // Where your server is located
+sets " Email"                                            ""   // Provide an email for technical support
+sets " Location"                                         ""   // Inform players of your server location
 
 //---------------------------------------------------------
 //  AUTHENTICATION SYSTEM SETTINGS
 //---------------------------------------------------------
 
-set  auth_enable                                        "1"   // Set this to 0 to disable the authentication system (more informations on www urbanterror info)
-set  auth_notoriety                                     "0"   // Minimum notoriety level to be able to connect to your server. 0 allows everyone to join
-set  auth_tags                                          "1"   // Set this to 1 to prevent clan tag thieves from joining your server
-set  auth_cheaters                                      "1"   // Set this to 1 to block the officially banned cheaters from your server [coming soon]
-set  auth_verbosity                                     "1"   // 0 = no authentication message to everyone when a player connects, 1 = message on the top of the screen, 2 = message in the chat box
-set  auth_log                                           "1"   // Set this to 1 to draw the account information in the userinfo of each player in the server logs
-set  auth_groups                                         ""   // Specify group IDs (separated by a blank) if you want to allow only a few groups to connect to the server. Empty ("") means all groups can connect
-seta auth_owners                                         ""   // Specify group IDs (separated by a blank) which can execute auth-rcon commands (more informations on www urbanterror info)
-
+sets auth_enable                                         "1"  // Set this to 0 to disable the authentication system (more information on www urbanterror info)
+sets auth_notoriety                                      "0"  // Minimum notoriety level to be able to connect to your server. 0 allows everyone to join
+sets auth_tags                                           "1"  // Set this to 1 to prevent clan tag thieves from joining your server
+sets auth_cheaters                                       "1"  // Set this to 1 to block the officially banned cheaters from your server [coming soon]
+sets auth_verbosity                                      "1"  // 0 = no authentication message to everyone when a player connects, 1 = message on the top of the screen, 2 = message in the chat box
+sets auth_log                                            "1"  // Set this to 1 to draw the account information in the userinfo of each player in the server logs
+sets auth_owners                                         "1"  // Seperate group IDs with a space which can execute auth-rcon commands (more information on www.urbanterror.info)
```

Comment:
Why would you give the FrozenSand group the right to use auth-rcon commands on all servers?